docset/winserver2022-ps/smbshare/Set-SmbServerConfiguration.md

```diff
@@ -730,6 +753,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -RequestCompression
+
+Indicates if SMB server should always request compression even if client or application didn't
+specify it.
```

Comment:
```suggestion
```suggestion
Indicates whether the SMB server should always request compression even if client or application
didn't specify it.
```
```

Reply:
Updated as suggested.